### DISCUSSION TITLE
Organization vs. organization unit

### DISCUSSION THREAD

**Discussion by rroupski** [ocsf/ocsf-schema#577] [Q&A]
The dictionary defines 3 attributes related to an organization: org_name, org_uid, and org_unit (see the descriptions on the picture below).

<img width="804" alt="image" src="https://user-images.githubusercontent.com/2128145/232871704-a2530ec1-d613-4af3-a990-17e17c81ab9a.png">

Based on the description example, it seems like the `org_name` is a company/business name. Is that correct-? If the answer is yes, then should we consider using the existing attribute `company_name`?

Since `org_name` name "pairs" with `org_uid`, is the `org_uid` the **uid** of the **organization** or **organization unit**? Perhaps we should use `org_unit_name` and `org_unit_uid`?

**Comment by pagbabian-splunk** [comment]
Yes, I just added org_name and changed the descriptions (in LDAP `org_name` is `o` and `org_unit` is `ou`.  I tried to roughly align with LDAP attribute descriptions for the same things.  However I also noticed that the `_uid` attribute only applies to one or the other.  There is no LDAP equivalent to the `_uid` attribute(s).

I want to make sure we can represent the Cloud platform org and account structures with an Org and Account object together.  The suggestions above are more explicit (i.e. `org_unit_uid`).

If we have an Org object, we would want to use `name` and `ou_name` along with `uid` and `ou_uid` though.

**Reply by irakledibm** [reply]
I would vote for Organization object. Potentially we may discover more attributes associated with an org that need to be added, it would be easier to have centralized control of those attributes if we have an object.

**Comment by pagbabian-splunk** [comment]
Thank you - I will create a PR for an `Organization` object with the 4 identified attributes and an `org` attribute.

**Comment by pagbabian-splunk** [comment]
The Organization object is part of RC3.

**Comment by irakledibm** [comment]
That is great!
